api-reference/docstubs/api/group-post-members.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Create members"
+description: "Create members by posting to the members collection."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Create members
+
+Namespace: microsoft.graph
+
+Create members by posting to the members collection.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /groups/{groupsId}/members/$ref
+POST /me/joinedGroups/{groupId}/members/$ref
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+|Content-Type|application/json.Required|
+
+## Request body
+In the request body, supply a JSON representation for the [directoryObject](../resources/directoryobject.md) object.
```

Comment:
OK - this isn't quite correct.
For adding a navigation (related link), the caller specifies the @odata.id property in the body.  So something like:
```
{
  "@odata.id": "https://graph.microsoft.com/beta/directoryObjects/{id}"
}
```
For an example see https://docs.microsoft.com/graph/api/application-post-owners?view=graph-rest-1.0&tabs=http
The table is wrong